.github/workflows/camunda-platform-release-main-dry-run.yml

```diff
@@ -20,7 +20,7 @@ jobs:
           - version: 0.8.2-alpha1
             latest: false
     with:
-      releaseBranch: main
```

Comment:
needed to test dry-run, I will revert it

Reply:
You could also use the other workflow https://github.com/camunda/zeebe/actions/workflows/camunda-platform-release-manual.yml and mark it as dryrun :)

Reply:
indeed